repo_id,repo_full_name,file,file_sha1,match
353538057,gerbsec/CVE-2020-24572-POC,exploit.py,a2cbce0782f87c5bdc3629f051400a09a4cf8dc3,CVE-2020-24572
353538057,gerbsec/CVE-2020-24572-POC,README.md,fa2e5cbe319731b98c569f4ce8cba3a43103cd58,CVE-2020-24572
353538057,gerbsec/CVE-2020-24572-POC,_GITHUB_REPO_METADATA_,,CVE-2020-24572
